subiquity/server/controllers/mirror.py

```diff
@@ -70,32 +57,20 @@ def load_autoinstall_data(self, data):
 
     @with_context()
     async def apply_autoinstall_config(self, context):
+        async def cc_is_set():
```

Comment:
This local function is pointless right? You can just put "asyncio.wait_for(self.cc_event.wait())" below?